openfecwebapp/config.py

```diff
@@ -5,7 +5,7 @@
 api_location_public = os.getenv('FEC_WEB_API_URL_PUBLIC', api_location)
 api_version = os.getenv('FEC_WEB_API_VERSION', 'v1')
 host = os.getenv('FEC_WEB_HOST', '0.0.0.0')
-port = os.getenv('VCAP_APP_PORT', '3000')
+port = os.getenv('FEC_WEB_PORT', '3000')
```

Comment:
Are we sure changing this isn't going to mess up CF? I recall `VCAP_APP_PORT` being a magical var name...